File: app/src/main/java/com/liweiyap/narradir/secrethitler/SecretHitlerCharacterName.java
package com.liweiyap.narradir.secrethitler;

/**
 * https://stackoverflow.com/questions/8157755/how-to-convert-enum-value-to-int
 */
public final class SecretHitlerCharacterName
{
    private SecretHitlerCharacterName(){}

    public static final int LIBERAL0 = 0;
    public static final int LIBERAL1 = 1;
    public static final int LIBERAL2 = 2;
    public static final int LIBERAL3 = 3;
    public static final int LIBERAL4 = 4;
    public static final int LIBERAL5 = 5;
    public static final int HITLER = 6;
    public static final int FASCIST0 = 7;
    public static final int FASCIST1 = 8;
    public static final int FASCIST2 = 9;

    public static int getNumberOfCharacters()
    {
        return 10;
    }

    public static int getDefaultNumberOfGoodCharacters() {
        return 3;
    }

    public static int getDefaultNumberOfEvilCharacters() {
        return 2;
    }
}

File: app/src/main/java/com/liweiyap/narradir/secrethitler/SecretHitlerCharacterName.kt
package com.liweiyap.narradir.secrethitler

object SecretHitlerCharacterName {
    const val LIBERAL0: Int = 0
    const val LIBERAL1: Int = 1
    const val LIBERAL2: Int = 2
    const val LIBERAL3: Int = 3
    const val LIBERAL4: Int = 4
    const val LIBERAL5: Int = 5
    const val HITLER: Int = 6
    const val FASCIST0: Int = 7
    const val FASCIST1: Int = 8
    const val FASCIST2: Int = 9

    @JvmStatic
    fun getNumberOfCharacters(): Int {
        return 10
    }

    @JvmStatic
    fun getDefaultNumberOfGoodCharacters(): Int {
        return 3
    }

    @JvmStatic
    fun getDefaultNumberOfEvilCharacters(): Int {
        return 2
    }
}